### ISSUE TITLE
Bug:  prettier 报错 ｜ Vben Drawer [warn]

### ISSUE THREAD

**Issue by elkon028** [vbenjs/vue-vben-admin#4385]
### Version

Vben Admin V5

## prettier 报错

```sh
Error: No package.json could be found upwards from directory .
    at Object.findRootSync (/Users/elkon/Desktop/wrokspace/vue-vben-admin/node_modules/.pnpm/@manypkg+find-root@2.2.3/node_modules/@manypkg/find-root/dist/manypkg-find-root.cjs.dev.js:130:11)
    at getPackagesSync (/Users/elkon/Desktop/wrokspace/vue-vben-admin/node_modules/.pnpm/@manypkg+get-packages@2.2.2/node_modules/@manypkg/get-packages/dist/manypkg-get-packages.cjs.dev.js:43:33)
    at getPackagesSync (/Users/elkon/Desktop/wrokspace/vue-vben-admin/internal/node-utils/src/monorepo.ts:29:43)
    at /Users/elkon/Desktop/wrokspace/vue-vben-admin/internal/tailwind-config/src/index.ts:16:49
    at evalModule (/Users/elkon/Desktop/wrokspace/vue-vben-admin/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/dist/jiti.js:1:247313)
    at jiti (/Users/elkon/Desktop/wrokspace/vue-vben-admin/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/dist/jiti.js:1:245241)
    at /Users/elkon/Desktop/wrokspace/vue-vben-admin/internal/tailwind-config/dist/index.cjs:7:3
    at evalModule (/Users/elkon/Desktop/wrokspace/vue-vben-admin/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/dist/jiti.js:1:247313)
    at jiti (/Users/elkon/Desktop/wrokspace/vue-vben-admin/node_modules/.pnpm/jiti@1.21.6/node_modules/jiti/dist/jiti.js:1:245241)
    at /Users/elkon/Desktop/wrokspace/vue-vben-admin/apps/web-ele/tailwind.config.mjs:1:337



```

<img width="1324" alt="image" src="https://github.com/user-attachments/assets/ee4a5b93-4639-465d-8530-4761ab4c4090">

## Vben Drawer [warn]
```log
[Vben Drawer]: When 'connectedComponent' exists, do not set props or slots 'class', which will increase complexity. If you need to modify the props of Drawer, please use useVbenDrawer or api.
```

### Reproduction

无

### System Info

```shell
System:
    OS: macOS 14.6.1
    CPU: (14) arm64 Apple M3 Max
    Memory: 149.45 MB / 36.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.16.0 - /usr/local/bin/node
    npm: 10.8.2 - /usr/local/bin/npm
    pnpm: 9.10.0 - /usr/local/bin/pnpm
    Watchman: 2024.09.02.00 - /opt/homebrew/bin/watchman
  Browsers:
    Chrome: 128.0.6613.138
    Safari: 17.6
```

### Validations

- [X] Read the [docs](https://anncwb.github.io/vue-vben-admin-doc/)
- [X] Ensure the code is up to date. (Some issues have been fixed in the latest version)
- [X] I have searched the [existing issues](https://github.com/vbenjs/vue-vben-admin/issues) and checked that my issue does not duplicate any existing issues.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vbenjs/vue-vben-admin/discussions) or join our [Discord Chat Server](https://discord.gg/8GuAdwDhj6).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by vince292007** [comment]
警告已经写得很清楚了，不要在外部的drawer加slot或者props，需要在usexxx内传递

**Comment by anncwb** [comment]
fixed #4391
